opgee/processes/gas_partition.py

```diff
@@ -89,7 +89,6 @@ def cache_attributes(self):
         self.GLIR = field.GLIR
         self.oil_volume_rate = field.oil_volume_rate
         self.WOR = field.WOR
-        self.iteration_tolerance = self.model.attr("iteration_tolerance")
```

Comment:
Wanted to double check that we don't want to add it as an xml attribute on the `GasPartition` process, specifically, a la:
```xml
<!-- attributes.xml -->
    <ClassAttrs name="GasPartition">
      <AttrDef name="iteration_tolerance" type="float" unit="frac" desc="...">0.000001</AttrDef>
      <!-- ... -->
    </ClassAttrs>
```
I fully support removing it from the configurable attributes if that's the intent.

Reply:
I asked Wennan about it and he didn't think it was something likely to be changed. It's pretty deep in the weeds of the gas partition process.